src/diffusion/include/antioch/molecular_binary_diffusion.h

```diff
@@ -122,10 +113,10 @@ namespace Antioch{
 
           //! \return molecular binary diffusion coefficient
           template <typename StateType>
-          const 
-          ANTIOCH_AUTO(StateType) 
-                binary_diffusion(const StateType & T, const StateType & cTot) const
-          ANTIOCH_AUTOFUNC(StateType,_coefficient * _interp.interpolated_value(T) / cTot )
+          const
+          ANTIOCH_AUTO(StateType)
+                binary_diffusion(const StateType & T, const StateType & molar_density) const
+          ANTIOCH_AUTOFUNC(StateType,_coefficient * _interp.interpolated_value(T) / molar_density )
 
            //!
```

Comment:
I know you didn't touch this, and maybe it's more suited to a separate doc pass, but this is an empty Doxygen-style comment.